pkg/controllers/provisioner/v1alpha1/allocation/greedyallocator.go

```diff
@@ -49,8 +49,8 @@ func (a *GreedyAllocator) Allocate(provisioner *v1alpha1.Provisioner, pods []*v1
 		if err != nil {
 			return fmt.Errorf("while creating capacity, %w", err)
 		}
-		for node, pods := range packing {
-			if err := a.bind(ctx, node, pods); err != nil {
+		for _, pack := range packing {
```

Comment:
Now that it's a list, I'd probably use the terms for _, packing := range packings.